notes/2023/2023-03-09.md

```diff
@@ -0,0 +1,53 @@
+# 2023-03-09 ESLint TSC Meeting Notes
+
+## Transcript
+
+
```

Comment:
The transcript format is always the same, so you can put it in and trust that it will be uploaded eventually. :)

```suggestion
[`2023-03-09-transcript.md`](2023-03-09-transcript.md)
```